test/data/testsuite.css

```diff
@@ -113,7 +113,7 @@ div#fx-tests div.noback {
 #nothiddendivchild.prct { font-size: 150%; }
 
 /* #9239 Attach a background to the body( avoid crashes in removing the test element in support ) */
-body, div { background: url(https://static.jquery.com/files/rocker/images/logo_jquery_215x53.gif) no-repeat -1000px 0; }
+body, div { background: url(1x1.jpg) no-repeat -1000px 0; }
```

Comment:
Does it work in both Karma & PHP modes? We generally prepend `baseURL` to paths which suggests such URLs are not enough, see e.g.:
https://github.com/jquery/jquery/blob/025da4dd343e6734f3d3c1b4785b1548498115d8/test/unit/css.js#L1134

Reply:
Ah yeah, the asserted value will need updating as well. Forgot about that.

Reply:
How come this is passing CI though?

https://github.com/jquery/jquery/blob/025da4dd343e6734f3d3c1b4785b1548498115d8/test/unit/css.js#L1553-L1571

Reply:
It seems the `expected` field is never read for anything, neither applied, nor compared with, nor displayed anywhere.

Reply:
I've revised the PR to remove this property.

Reply:
The reason for the `1x1.jpg` background is outlined in the bug mentioned above: https://bugs.jquery.com/ticket/9239. Apparently, in the past we've created an additional hidden body to perform support tests in and this was causing some bugs in some browsers. The issue was fixed in https://github.com/jquery/jquery/commit/ceba855c010c792aad8fc15edc06b86285f71142 and this background was added in https://github.com/jquery/jquery/commit/e5457a5a1faa642b0af13e87ee3f76f2ef704b30.

We already have the test for that in `bodyBackground.html` so I guess this background was meant to just avoid some crashes in tests, perhaps related to old IE? In that case, it should be possible to just remove it. On real sites all divs wouldn't have background set anyway so removing it would get us closer to reality.

@timmywil since you added this, any objections to removing this block?

Reply:
All the way back in 2011. Yea, it's probably fine to remove now.